meetup/migrations/0001_initial.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.2 on 2016-03-04 15:03
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import meetup.models
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+        ('pizzaplace', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Meetup',
+            fields=[
```

Comment:
Something I've learned the hard way: Django models don't provide time stamps by default! The most common convention is to use [this abstract base class](https://django-model-utils.readthedocs.org/en/latest/models.html#timestampedmodel) instead of django.db.models.Model as your default model class. Note that it comes from a third-party library! :scream_cat: